src/components/SearchFilters.vue

```diff
@@ -124,251 +161,429 @@ export default {
       showFilters: true,
       showFiltersText: true,
       cascadeSelected: [],
-      cascadeSelectedWithBoolean: [], 
+      cascadeSelectedWithBoolean: [],
       numberShown: 10,
       filters: [],
-      facets: ["Species", "Gender", "Organ", "Datasets"],
-      numberDatasetsShown: ["10", "20", "50"],
+      facets: ['Species', 'Gender', 'Organ', 'Datasets'],
+      numberDatasetsShown: ['10', '20', '50'],
       props: { multiple: true },
       options: [
         {
-          value: "Species",
-          label: "Species",
+          value: 'Species',
+          label: 'Species',
           children: [{}],
         },
       ],
-    };
+      cascaderTags: {},
+      presentTags:[],
+      correctnessCheck: {
```

Comment:
This can be implemented using markRaw as it does not need to be reactive.

Reply:
I will update this.